ewapt.py

```diff
@@ -469,6 +474,20 @@ def getPriceBase(cmd):
 
 	furns = ewitem.inventory(id_user= lookObject+ewcfg.compartment_id_decorate, id_server= playermodel.id_server, item_type_filter=ewcfg.it_furniture)
 
+	furniture_name_list = []
+	for furnName in furns:
+		furniture_name_list.append(furnName.get('name'))
+
+	if all(elem in furniture_name_list for elem in ewcfg.furniture_lgbt):
```

Comment:
I don't like that we're using string names for these comparisons. that's what id_furniture should be for. string names aren't guaranteed to be unique.